packages/mongo/mongo_driver.js

```diff
@@ -1106,6 +1106,26 @@ _.extend(SynchronousCursor.prototype, {
   }
 });
 
+SynchronousCursor.prototype[Symbol.iterator] = function () {
+  var self = this;
+
+  // Get back to the beginning.
+  self._rewind();
+
+  return {
+    next: function () {
+      var doc = self._nextObject();
+
+      if (!doc) {
+        return {done: true};
+      }
+      else {
```

Comment:
Since you're returning in the case above, you can avoid the `else` here and just `return { value: doc, done: false }` below.

Reply:
Updated.